Box.V2/Models/Request/BoxApplyWatermarkRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Box.V2.Models
+{
+    public class BoxApplyWatermarkRequest
+    {
+        private BoxWatermarkRequest watermark = new BoxWatermarkRequest();
```

Comment:
hmm, reading through all this now.  I'm going to need to pull this PR down and take a look at it in detail